### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,96 +27,154 @@ Or install it yourself as:
...
+```
 
 For testing and development purpose you can disable the ssl check with:
 
-`client.ssl_options(verify_ssl: OpenSSL::SSL::VERIFY_NONE)`
+```ruby
+client.ssl_options(verify_ssl: OpenSSL::SSL::VERIFY_NONE)
+```
+
+If you are using bearer tokens for authentication as described
```
Comment (abonas) [inline README.md]: this belongs to a different PR that wasn't merged yet :)

File: README.md [hunk 2]
```diff
@@ -27,96 +27,154 @@ Or install it yourself as:
...
+service.spec.port = 6379
+service.spec.containerPort  = "redis-server"
+service.spec.selector = {}
+service.spec.selector.name = "redis"
+service.spec.selector.role = "master"
+client.create_service service`
+```
+
+#### Update entity
+And respectively `update_pod`, `update_service`, update_replication_controller`
```
Comment (abonas) [inline README.md]: missing  ` char at beginning of "update_replication_controller"

**Comment by abonas** [comment]
@jimmidyson , reviewed, please check comments.
in overall  - :+1: 
thanks for this!

**Comment by jimmidyson** [comment]
@abonas Updated with your comments. If you merge this I'll update the bearer token PR with details in the README on that PR

**Comment by abonas** [comment]
:+1:
